content/spire/getting-started-linux.md

```diff
@@ -91,25 +91,12 @@ To install the server and agent:
 2. Add `spire-server` and `spire-agent` to your $PATH for convenience:
 
 	```shell
-	ln -s /opt/spire/spire-server /usr/bin/spire-server
-	ln -s /opt/spire/spire-agent /usr/bin/spire-agent
+	sudo ln -s /opt/spire/spire-server /usr/bin/spire-server
+	sudo ln -s /opt/spire/spire-agent /usr/bin/spire-agent
```

Comment:
Should this be /opt/spire/bin/spire-server and /opt/spire/bin/spire-agent ?

Reply:
Yes, great catch.